src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -144,13 +144,14 @@ static async Task<char> GetInsertionCharacterAsync(Document document, int positi
         public static Uri GetUriFromFilePath(string? filePath)
```

Comment:
should this be replaced entirely by TryGetUriFromFilePath?

Reply:
makes sense.